lasagne/layers/dense.py

```diff
@@ -89,6 +88,35 @@ def get_output_for(self, input, **kwargs):
         return self.nonlinearity(activation)
 
 
+class NonlinearityOnlyLayer(Layer):
+    """
+    A layer that just performs a nonlinearity.
+    This is for use in conjunction with MultipleInputsLayer subclasses,
+    which just add or concatenate outputs from layers.  For example, one
+    might want to take inputs of length n and m, perform linear transformations
+    on them using DenseLayer objects, then add the transformed outputs with an
+    ElemwiseSumLayer, and then use this class to apply a nonlinearity
+    (note that this would be different from applying the nonlinearity and
+    then adding).  Alternatively, one could use a ConcatLayer followed by
+    a DenseLayer, but concatenation is less efficient than adding.
+
+    - input_layer : `Layer` instance
+        The layer from which this layer will obtain its input
```

Comment:
It should be `incoming`. You can just copy that from the base `Layer` docstring.